src/Router.cpp

```diff
@@ -10,3 +13,15 @@ Router::current() const
 {
   return current_page;
 }
+
+// ------- //
+// Méthode //
+// ------- //
+
+Router&
+Router::go(const Page page, const int encoder)
+{
+  current_page = page;
+
+  rotary_encoder.set_position(encoder);
+}
```

Comment:
Oops, le retour de cette fonction est incorrect.
```suggestion
Router&
Router::go(const Page page, const int encoder)
{
  current_page = page;

  rotary_encoder.set_position(encoder);
  
  return *this;
}
```

Reply:
De plus, la définition dans l'en-tête est manquante.